### PR TITLE
Implement the Java NodeTypes

### PR DESCRIPTION
**Pull request by mklopp**

### CODE REVIEW

**Comment by hnicke** [comment]
Is this still WIP or ready fro review?

**Comment by mklopp** [comment]
This PR isn't ready for review. But nobody is working on it right now so i would say it is iceboxed.

**Comment by hnicke** [comment]
Is it reasonable to assume the aws plugin will support the java csar in the end?

**Comment by mklopp** [comment]
works

File: server/src/main/java/org/opentosca/toscana/plugins/cloudformation/mapper/JavaRuntimeMapper.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public JavaRuntimeMapper(Logger logger) {
+        this.logger = logger;
+    }
+
+    public String mapRuntimeToStackConfig(JavaRuntime javaRuntime) {
+        logger.debug("Map JavaRuntime {}", javaRuntime.getEntityName());
+        String javaVersion;
+        if (javaRuntime.getComponentVersion().isPresent()) {
+            javaVersion = javaRuntime.getComponentVersion().get().replaceAll("1.", "");
```
Comment (hnicke) [inline server/src/main/java/org/opentosca/toscana/plugins/cloudformation/mapper/JavaRuntimeMapper.java]: Be warned: `replaceAll` takes a regex. I bet youmeant to write: `replaceAll("1\\.", "");`

But still, reminds me of the Y2K bug (java's latest version is already 1.9) . Consider:
```java
...replaceAll("^.*\\.","")
```

Comment (mklopp) [reply server/src/main/java/org/opentosca/toscana/plugins/cloudformation/mapper/JavaRuntimeMapper.java]: Totally missed that

**Comment by mklopp** [comment]
Changed the regex. I'd like to have @mmunozba take a look at this before merging.

**Pull request merged**